### PR TITLE
Use INFORMATION_SCHEMA system table in mysql2 adapter

### PR DESCRIPTION
**Pull request by abicky**
`SELECT COUNT(*)` on large tables takes too much time, so use INFORMATION_SCHEMA system table, which has the approximate value.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/repro
```
Comment (hogelog) [inline .circleci/config.yml]: 😃

Comment (abicky) [reply .circleci/config.yml]: oh...

**Comment by hogelog** [comment]
CircleCI migration is merged at https://github.com/hogelog/dmemo/pull/115 👍 @abicky

File: spec/models/data_source_adapters/mysql2_adapter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe DataSourceAdapters::Mysql2Adapter, type: :model do
```
Comment (hogelog) [inline spec/models/data_source_adapters/mysql2_adapter_spec.rb]: > __[RuboCop]__ Metrics/BlockLength: Block has too many lines. [28/25] ([link](https://sider.review/gh/repos/55787616/pulls/114#issue-3762142))

I don't like this rule 😊

Comment (hogelog) [reply spec/models/data_source_adapters/mysql2_adapter_spec.rb]: https://github.com/hogelog/dmemo/commit/64ac8ef7996bdc70de32d368933252042e9eae7c disabled

**Comment by abicky** [comment]
Thank you for your review!

**Pull request merged**